superset/examples/configs/datasets/examples/unicode_test.yaml

```diff
@@ -0,0 +1,77 @@
+table_name: unicode_test
+main_dttm_col: dttm
+description: null
+default_endpoint: null
+offset: 0
+cache_timeout: null
+schema: null
+sql: null
+params: null
+template_params: null
+filter_select_enabled: false
+fetch_values_predicate: null
+extra: null
+uuid: a6771c73-96fc-44c6-8b6e-9d303955ea48
+metrics:
+- metric_name: count
+  verbose_name: COUNT(*)
+  metric_type: count
+  expression: COUNT(*)
+  description: null
+  d3format: null
+  extra: null
+  warning_text: null
+columns:
+- column_name: with_missing
+  verbose_name: null
+  is_dttm: false
+  is_active: true
+  type: VARCHAR(100)
+  groupby: true
+  filterable: true
+  expression: ''
+  description: null
+  python_date_format: null
+- column_name: phrase
+  verbose_name: null
+  is_dttm: false
+  is_active: true
+  type: VARCHAR(500)
+  groupby: true
+  filterable: true
+  expression: ''
+  description: null
+  python_date_format: null
+- column_name: short_phrase
+  verbose_name: null
+  is_dttm: false
+  is_active: true
+  type: VARCHAR(10)
+  groupby: true
+  filterable: true
+  expression: ''
+  description: null
+  python_date_format: null
+- column_name: dttm
+  verbose_name: null
+  is_dttm: true
+  is_active: true
+  type: DATE
+  groupby: true
+  filterable: true
+  expression: ''
+  description: null
+  python_date_format: null
+- column_name: value
+  verbose_name: null
+  is_dttm: false
+  is_active: true
+  type: FLOAT
+  groupby: true
+  filterable: true
+  expression: ''
+  description: null
+  python_date_format: null
+version: 1.0.0
+database_uuid: a2dc77af-e654-49bb-b321-40f6b559a1ee
+data: https://gist.githubusercontent.com/betodealmeida/bd3b49ec1e2ee230e56854cf1b72a131/raw/7cf28e3e71250e7ba7443a2b692f6a2d694b26d6/unicode_test_data.csv
```

Comment:
I need to update this to a URL in https://github.com/apache-superset/examples-data/blob/master/